sys/include/net/gnrc/rpl.h

```diff
@@ -82,6 +82,42 @@
  *   CFLAGS += -DCONFIG_GNRC_RPL_WITHOUT_VALIDATION
  *   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  *
+ * - This RPL implementation currently only supports storing mode.
+ *   That means, in order to have downwards routes to all nodes the
+ *   Neighbor Information Base must be big enough to store information for
```

Comment:
What is a Neighbor Information Base? ;-)

Reply:
You know better than me ;)

What do you suggest to put here? After all, those tunables are very much RIOT specific.

I would very much welcome a glossary in the documentation. I've scratched my head more than once what those TLAs in the network stack all *mean*.

Reply:
IMHO it is enough to add a link here.

```
 *   That means, in order to have downwards routes to all nodes the
 *   storage space within `gnrc_ipv6`'s [Neighbor Information Base](@ref net_gnrc_ipv6_nib)
 *   must be big enough to store information for each node.
```

Reply:
The term NIB is an invention of mine, as it is not just a neighbor cache (that's why the N stands for "Neighbor") but also more than a FIB. The comparison I always draw is Linux's `iproute2` which is also handling the ARP tables and the IPv6 neighbor cache on Linux.